pkg/operator/injection/injection.go

```diff
@@ -71,7 +86,7 @@ func GetNodeClasses(ctx context.Context) []schema.GroupVersionKind {
 	retval := ctx.Value(nodeClassKey)
 	if retval == nil {
```

Comment:
You could just skip this check. It'll be clear from the trace where/why this crashed.